apps/bwclk/app.js

```diff
@@ -270,6 +325,14 @@ let lockListenerBw = function(isLocked) {
 };
 Bangle.on('lock', lockListenerBw);
 
+let charging = function(charging){
+  // Jump to battery
+  clockInfoMenu.menuA = 0;
+  clockInfoMenu.menuB = 2;
+  clockInfoMenu.redraw();
+  drawTime();
+}
+Bangle.on('charging', charging);
```

Comment:
`Bangle.removeListener('charging', charging);` needs to be added to the `remove` function in setUI to not get ram leak.

Reply:
Oh wow I was not aware of that! Its fixed now, thanks a lot!

Reply:
Feel free to unresolve in case something is still wrong :)

Reply:
Looks good i think!

@gfwilliams has provided a tutorial regarding fast loading [here](https://www.espruino.com/Bangle.js+Fast+Load) :)